### ISSUE TITLE
Training the Neural Network

### ISSUE THREAD

**Issue by cormacmchale** [cormacmchale/KerasNeuralNetwork#2]
Need to research this

**Comment by cormacmchale** [comment]
https://www.youtube.com/watch?v=EHbay6hDZGA
using this as a reference

**Comment by cormacmchale** [comment]
for the moment I will be basing my initial building of the network on the jupyter notebook that our lecturer has provided us with.
https://nbviewer.jupyter.org/github/ianmcloughlin/jupyter-teaching-notebooks/blob/master/mnist.ipynb

**Comment by cormacmchale** [comment]
https://keras.io/activations/
Might be something here for training the Network better?

**Comment by cormacmchale** [comment]
model.summary()
Gives a breakdown of the model.. could be useful later on??
![Capture](https://user-images.githubusercontent.com/33293951/67144004-99f57900-f269-11e9-8a44-dd254ad1864a.PNG)

**Comment by cormacmchale** [comment]
https://www.youtube.com/watch?v=Skc8nqJirJg
Some extra explaining of the loss function

**Comment by cormacmchale** [comment]
Playing around with the different variables in the keras library.. the model so far has become this accurate..
![Capture](https://user-images.githubusercontent.com/33293951/67146348-198f4200-f282-11e9-80cb-49e7f1666655.PNG)
will have to run this against the test images now

**Comment by cormacmchale** [comment]
The basic training and testing of a model implemented
![Capture](https://user-images.githubusercontent.com/33293951/67147510-6e848580-f28d-11e9-846c-ed84e6d9be5f.PNG)
Will just need to do more research on the model functions to make a better model....
